home/templatetags/generic_components.py

```diff
@@ -84,3 +88,47 @@ def language_picker_style():
 @register.simple_tag
 def site_version():
     return settings.SITE_VERSION
+
+
+@register.simple_tag(takes_context=True)
+def google_analytics(context, tracking_code=None, debug=False):
```

Comment:
I must assume that this is original code that hasn't been lifted from another project. In any case, this code should be fairly easy to write unit tests for and I would like to see some, please.

Reply:
added test cases